app/components/Thread.tsx

```
@@ -126,54 +162,67 @@ export default function Post({
-      <div className={styles.post}>
+    <article key={_id} className={styles.threadWrapper}>
+      <div
+        className={styles.thread}
+        onClick={handleThreadClick}
+        onKeyDown={handleThreadKeyDown}
+        role={variant === "feed" ? "button" : undefined}
+        tabIndex={variant === "feed" ? 0 : undefined}
+        aria-label={
+          variant === "feed" ? `View thread by ${thread.author?.name}` : undefined
+        }
+      >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)